jscomp/stubs/ext_basic_hash_stubs.c

```diff
@@ -3,6 +3,7 @@
 #include <string.h>
 #include <stdint.h>
 #include "caml/memory.h"
+#include "caml/osdeps.h"
```

Comment:
Any idea of what happened?
Was this removed or was it never there?

Reply:
In 9.1.4, this wasn't there either, but
```c
  ret = CAML_SYS_STAT(p, &st);
```
was used instead of
```c
  ret = stat_os(p, &st);
```

Reply:
(Now `CAML_SYS_STAT` does not exist anymore, this was probably an OCaml 4.06 thing?)